set.mm

```diff
@@ -617513,6 +617517,26 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
       GKJZLMUDBRZUFUILUJUEUGUCKUDBHIOPQUCAUGLKSDCTUHLUAUB $.
   $}
 
+  $(
+    TODO: The proof is that <. a , b , c , ... >. has either a = 0 or a =/= 0.
+    Since <. 0 , b , c , ... >. .~ <. 0 , nb, nc , ... >., the case a = 0
+    corresponds to the (n-1)-dimensional projective space. When a =/= 0, there
+    is <. a , b , c , ... >. .~ <. 1 , b/a, c/a, ... >.  Since the later terms
+    are irreducible it corresponds to all (n-1)-tuples of ( Base ` K ) which is
+    equivalent to the construction of an affine space.  Note that the closest
+    definition to affine space so far seems to be ~ df-ehl , which is specific
+    to reals.
+
+    prjspnf1om1.1 $e |- F = ??? $.
+    @( A bijection between an n-dimensional projective space and its
+       (n-1)-dimensional affine and projective spaces.  (Contributed by Steven
+       Nguyen, ??-??-2023.) @)
+    prjspnf1om1 @p |- ( ( N e. NN /\ K e. DivRing /\ ?? ) ->
+      F : ( N PrjSpn K ) -1-1-onto-> ( ( K ^m ( 0 ..^ ( N - 1 ) ) ) u.
+                                       ( ( N - 1 ) PrjSpn K ) ) ) @=
+      ? $.
+  $)
```

Comment:
`~f1oun` seems to be the best top-level theorem to start with.
Then you can probably explicitly construct the mappings.